src/services/util.js

```diff
@@ -1,7 +1,22 @@
+import {
+  getLocalStorage,
+  TOKEN_KEY,
+  USER_DATA_KEY,
+  isBrowser,
+} from './localStorage'
 export const getYoutubeThumbnail = url => {
   if (url && url.match(/youtube/)) {
     const id = url.match(/v=(.+)$/)[1]
 
     return `https://i.ytimg.com/vi/${id}/mqdefault.jpg`
   }
 }
+
+export const isLoggedIn = () => {
+  return getLocalStorage(TOKEN_KEY) ? true : false
+}
+
+export const getUser = () =>
+  isBrowser && window.localStorage.getItem(USER_DATA_KEY)
```

Comment:
no need to repeat `window.localStorage.getItem(USER_DATA_KEY)` twice .. put it in variable